lib/utils/timetable.ts

```diff
@@ -0,0 +1,34 @@
+import { DevelopmentPlanTimetable, FormData } from "../types/timetable";
+import { v4 as uuidv4 } from "uuid";
+
+const formStateToDevelopmentPlanTimetables = (
```

Comment:
shall we add unit tests for these 2 functions as well?

Reply:
I started but they end up being kind of long for two very simple functions